content-src/components/Rating/Rating.js

```diff
@@ -0,0 +1,45 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {justDispatch} = require("selectors/selectors");
+const {actions} = require("common/action-manager");
+const classNames = require("classnames");
+
+const Rating = React.createClass({
+  getInitialState() {
+    return {numStars: 5, activeRatings: {}, currentRating: null};
```

Comment:
make `numStars` a prop and instead of using `activeRatings` just check if your index is less than `currentRating`